zephyr/CMakeLists.txt

```diff
@@ -1,6 +1,75 @@
 if(CONFIG_BOOTLOADER_MCUBOOT)
-  # Build a second bootloader image
+  function(sign to_sign_hex output_prefix offset sign_depends signed_hex_out)
```

Comment:
Could this function be upstreamed?

Reply:
Without multi image I don't see any benefit of this, as the user would have to manually point to what hex file should be signed etc.

Also, upstream doesn't have mcuboot as a west-module, hence this file does not exist upstream.